src/routes/common/pagination/pagination.data.spec.ts

```diff
@@ -133,7 +133,7 @@ describe('PaginationData', () => {
       const expected = new URL(
         'https://base.url/?cursor=limit%3D10%26offset%3D2',
       );
-      expect(actual).toStrictEqual(expected);
+      expect(actual?.href).toStrictEqual(expected.href);
```

Comment:
I had to change this because it seems Node 18.17 makes stricter comparisons between objects here. The part we are interested in is the URL's `href`, so I'm comparing both `href` properties. Probably this was raised because of the Node.js update to its URL parser (from [Changelog](https://raw.githubusercontent.com/nodejs/node/main/doc/changelogs/CHANGELOG_V18.md)):

```
Node.js v18.17.0 comes with the latest version of the URL parser, Ada. This update brings significant performance improvements
to URL parsing, including enhancements to the url.domainToASCII and url.domainToUnicode functions in node:URL.
```

But in any case, if I'm not wrong, `toStrictEqual` compares the whole object prototype, so I'm not sure why this was even working before.